tests/src/common/http_resumer/http_resumer_test.cpp

```diff
@@ -104,11 +104,12 @@ class RangeBodyOfXes : virtual public io::Reader {
 	}
 
 	static uint8_t TransferFunction(size_t index) {
-		// Fill in a specific pattern to try to catch offset/block errors: Raise the input
-		// number to the power of 1.1 and round to the nearest integer. If it's odd, return
-		// 'X', if it's even, return 'x'. Due to the exponent, this pattern will change
-		// slightly throughout the sequence, giving us a chance to catch offset errors.
-		// (Note: Try printing it, the pattern is mesmerizing to watch!)
+		// Fill in a specific pattern to try to catch offset/block errors: Raise the
+		// input number to the power of 1.1 and round to the nearest integer. If
+		// it's odd, return 'X', if it's even, return 'x'. Due to the exponent, this
+		// pattern will change slightly throughout the sequence, giving us a chance
+		// to catch offset errors. (Note: Try printing it, the pattern is
+		// mesmerizing to watch!)
```

Comment:
This file has a lot of unrelated changes. Please remove them from this PR. (If you really want to change the style of the comments, at lest do so on a separate commit).

Tip :bulb: 

I assume you got into this because your editor modifies the line with of the code and later you did `git add <file>` or even `git commit -a`. I recommend using `git add -i` (for interactive) so that you can state patch by patch which changes you want to add to your next commit :+1:

Reply:
Oh sorry, this was because the formatting tests complained about this files formatting, but must have formatted it wrongfully, will fix this. Thanks for the tip :)